content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -0,0 +1,72 @@
+<li>Schedule CHW monthly meetings quality monitoring visits and follow up visits.</li>
+<li>Support supervisors to report on health outcomes for their supervisory area.</li>
+<li>Identify CHWs who have not been visited by a supervisor.</li>
+<li>Follow up on CHW who have been inactive for 3 months.</li>
+</ul>
+
+## Forms hierarchy
+The diagram indicates forms and tasks that can be filled by a supervisor on the supervisor reference app. Some of the forms are accessible as actions on the supervisor and CHW profiles while others are accessible as tasks for a supervisor and a CHW.
+
+{{< figure src="Forms_hierarchy.png"  link="Forms_hierarchy.png" alt="Forms hierarchy" title="" class="left col-12">}}
+
+## Supervisor app workflows and features.
+
+### CHW monthly meeting workflows.
+A Supervisor conducts a CHW monthly meeting with all the CHWs. During the meeting the supervisor mentors the CHWs, reviews the CHWs performance and discusses with the CHWs the challenges they might be facing in their provision of community health services.
+
+{{< figure src="Monthly_meeting_workflows.png"  link="Monthly_meeting_workflows.png" alt="Monthly meeting workflows" title="" class="left col-10">}}
+
+### Quality monitoring workflow
+With the supervisor reference app, a supervisor can schedule a quality monitoring visit to assess the quality of services being provided by a CHW. During the quality monitoring visit a supervisor shadows CHWs as they provide services to household members. A supervisor can schedule additional quality monitoring follow up visits for CHWs who are identified to have some weak areas and may need further support and mentorship from the supervisor.
+
+{{< figure src="Qualtiy_monitoring_workflows.png"  link="Qualtiy_monitoring_workflows.png" alt="Qualtiy monitoring workflows" title="" class="left col-10">}}
+
+### Group session workflow
+
+{{< figure src="Group_session_workflows.png"  link="Group_session_workflows.png" alt="Group session workflows" title="" class="left col-10">}}
+
+### CHW Activity workflow
+A CHW activity task is generated for a supervisor if a CHW has been inactive for 3 months. This workflow enables a supervisor to follow up CHWs who have been inactive for three months and may need to be replaced.
+
+### Supervisor in-app targets
+A Supervisor has access to the analytics tab. The widgets on the in-app analytics tab enables a supervisor to view individual supervisor performance metrics and the supervisor progress towards the expected monthly goals. The supervisor also has access to CHW aggregate targets which will help the supervisor track metrics for individual CHWs, especially for supervisors overseeing a group of CHWs.This feature allows a supervisor to identify CHWs who may need to be supported actively.
+
+##### CHW aggregate targets
+{{< figure src="CHW_aggregate_targets.png"  link="CHW_aggregate_targets.png" alt="CHW aggregate targets" title="" class="left col-10">}}
```

Comment:
be sure  images don't have "rectangular snip" ghost image:

![image](https://user-images.githubusercontent.com/8253488/194178013-8a572d00-23b8-4ef5-9e00-1dc352724684.png)